### PR TITLE
[MTM-49354]: Added announcement about a breaking change introduced fo…

### PR DESCRIPTION
**Pull request by c8y-kh4mell**
…r SmartREST 2.0

### CODE REVIEW

File: content/release-10-16-0/announcements-10-16-0.md [hunk 1]
```diff
@@ -8,6 +8,60 @@ layout: bundle
...
+
+New format:
+
+```json
+{
+   "source":{
+      "id":"91123"
+   },
+   "type":"TYPE-911",
+   "date":"2022-11-14T12:44:11.481+01:00",
```
Comment (slc-softwareag) [inline content/release-10-16-0/announcements-10-16-0.md:54]: Maybe rename the `date` field to something like `myCustomDate` so it is not mixed with our standard time fields

**Review by slc-softwareag — APPROVED** [review]

**Review by jkot-softwareag — APPROVED** [review]

File: content/release-10-16-0/announcements-10-16-0.md (hunk 1 above)
Comment (BeateRixen) [reply content/release-10-16-0/announcements-10-16-0.md]: @c8y-kh4mell @martin-sag Breaking changes must be announced two release upfront, so this should be added to the 10.15 announcements as well, shouldn´t it? Still possible as they are still in preview mode.

File: content/release-10-16-0/announcements-10-16-0.md [hunk 2]
```diff
@@ -8,6 +8,60 @@ layout: bundle
 
 #### Planned
 
+##### Breaking change in the SmartREST 2.0 -- a DATE field used as a custom property will be stored as a String.
```
Comment (BeateRixen) [inline content/release-10-16-0/announcements-10-16-0.md:11]:
```suggestion
##### Breaking change in SmartREST 2.0 - DATE field used as custom property will be stored as string
```

File: content/release-10-16-0/announcements-10-16-0.md [hunk 3]
```diff
@@ -8,6 +8,60 @@ layout: bundle
...
+With 10.17 release a SmartREST 2.0 DATE field used as a custom property will be stored as a String in the Cumulocity database.
```
Comment (BeateRixen) [inline content/release-10-16-0/announcements-10-16-0.md:12]:
```suggestion
As of release 10.17, a SmartREST 2.0 DATE field used as a custom property will be stored as a string in the Cumulocity IoT database.
```

File: content/release-10-16-0/announcements-10-16-0.md [hunk 4]
```diff
@@ -8,6 +8,60 @@ layout: bundle
...
+This affects REST response format.
```
Comment (BeateRixen) [inline content/release-10-16-0/announcements-10-16-0.md:13]:
```suggestion
This affects the REST response format.
```

File: content/release-10-16-0/announcements-10-16-0.md [hunk 5]
```diff
@@ -8,6 +8,60 @@ layout: bundle
...
+   "aCustomDateField":"2022-11-14T12:44:11.481+01:00",
+   "c8y_TemperatureMeasurement":{
+      "T":{
+         "unit":"C",
+         "value":10
+      }
+   }
+}
+```
+The change is introduced to improve a consistency between different Cumulocity protocols.
```
Comment (BeateRixen) [inline content/release-10-16-0/announcements-10-16-0.md:63]:
```suggestion
The change will be introduced to improve the consistency between different Cumulocity IoT protocols.
```

**Pull request merged**